domain/src/main/java/org/oppia/domain/oppialogger/OppiaLogger.kt

```diff
@@ -0,0 +1,123 @@
+package org.oppia.domain.oppialogger
+
+import org.oppia.app.model.EventLog
+import org.oppia.app.model.EventLog.EventAction
+import org.oppia.domain.oppialogger.analytics.AnalyticsController
+import javax.inject.Inject
+
+/** Logger that handles exceptions, crashes, events, and console logging. */
```

Comment:
Suggest also mentioning that this logger should be used in favor of all other loggers.

Reply:
Done